app/_how-tos/authenticate-consumers-with-key-auth-enc.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Enable key authentication on a Gateway Service with {{site.base_gateway}}
+content_type: how_to
+
```

Comment:
```suggestion
description: Secure a Gateway Service with the Key Auth Encrypted plugin.
```
Feel free to adjust however you want, but we should have `description` on all how-tos now.